trigger/task.go

```diff
@@ -492,10 +489,6 @@ func (t *taskTriggers) taskFirstFailureInVersion(sub *event.Subscription) (*noti
 }
 
 func (t *taskTriggers) taskFirstFailureInVersionWithName(sub *event.Subscription) (*notification.Notification, error) {
-	if t.task.DisplayOnly {
```

Comment:
for the remaining cases where we will notify for execution tasks does it still sense for [makeData](https://github.com/evergreen-ci/evergreen/blob/846380647d5f1cfe167cc711a64be8b8b4c756a4/trigger/task.go#L263) to switch the status and name? It looks like the reason Richard did it no longer applies (https://github.com/evergreen-ci/evergreen/pull/1596#issue-211498813)

Reply:
I'm not sure this question is addressed: we intend to notify on the execution task. Why do we switch the notification to be about the display task? 
For example, when the first failure in a version is an execution task and other tasks in the display task are still running, won't the user get a notification about that their started display task was the first failure in the version?

Reply:
Ah, I think I understand: users are supposed to think of display tasks. Still, it's confusing to get a notification about a started display task that caused the version to fail.

Reply:
Good point. I made certain triggers use the execution task status if available